equiv_yosys.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+#echo "Comparing $1 and $2 with $3"
+yosys -q -p "
```

Comment:
Why not run yosys directly on a tcl script? Lit is perfectly capable of detecting exit code failures.